[
   {
      "body" : "These are disk corruption issues: leveldb keeps its own checksums of all data it writes, when it notices during (background) compaction that these mismatch it will return the error on the next flush.\r\n```\r\n2015-08-06 05:53:59 Corruption: block checksum mismatch\r\n2015-08-06 05:53:59 *** System error while flushing: Database corrupted\r\n```",
      "created_at" : "2015-08-07T06:18:36Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-128609395",
      "id" : 128609395,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-08-07T06:19:19Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/128609395",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/126646?v=3",
         "events_url" : "https://api.github.com/users/laanwj/events{/privacy}",
         "followers_url" : "https://api.github.com/users/laanwj/followers",
         "following_url" : "https://api.github.com/users/laanwj/following{/other_user}",
         "gists_url" : "https://api.github.com/users/laanwj/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/laanwj",
         "id" : 126646,
         "login" : "laanwj",
         "organizations_url" : "https://api.github.com/users/laanwj/orgs",
         "received_events_url" : "https://api.github.com/users/laanwj/received_events",
         "repos_url" : "https://api.github.com/users/laanwj/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/laanwj/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/laanwj/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/laanwj"
      }
   },
   {
      "body" : "Same issue here, but with Win10 x64. Checkdisk returned no errors on disk.",
      "created_at" : "2015-08-11T19:37:32Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-130035599",
      "id" : 130035599,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-08-11T19:38:03Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/130035599",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/3930743?v=3",
         "events_url" : "https://api.github.com/users/Weax/events{/privacy}",
         "followers_url" : "https://api.github.com/users/Weax/followers",
         "following_url" : "https://api.github.com/users/Weax/following{/other_user}",
         "gists_url" : "https://api.github.com/users/Weax/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/Weax",
         "id" : 3930743,
         "login" : "Weax",
         "organizations_url" : "https://api.github.com/users/Weax/orgs",
         "received_events_url" : "https://api.github.com/users/Weax/received_events",
         "repos_url" : "https://api.github.com/users/Weax/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/Weax/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/Weax/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/Weax"
      }
   },
   {
      "body" : "The support team of the VPS hosting provider claims there are no disk corruption issues:\r\n\r\n```\r\nThere is no disk corruption from our side, because all our nodes use RAID array in which case\r\ndisks is duplicated in the array and in case of any disk failure this disk is automatically removed\r\nfrom the array and node is working with other left disk until faulty hard disk replace.\r\nYou database corruption may be related to your program/service crash or errors in it etc.\r\n```",
      "created_at" : "2015-08-13T10:11:16Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-130602302",
      "id" : 130602302,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-08-13T10:11:29Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/130602302",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/620959?v=3",
         "events_url" : "https://api.github.com/users/jorisbontje/events{/privacy}",
         "followers_url" : "https://api.github.com/users/jorisbontje/followers",
         "following_url" : "https://api.github.com/users/jorisbontje/following{/other_user}",
         "gists_url" : "https://api.github.com/users/jorisbontje/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/jorisbontje",
         "id" : 620959,
         "login" : "jorisbontje",
         "organizations_url" : "https://api.github.com/users/jorisbontje/orgs",
         "received_events_url" : "https://api.github.com/users/jorisbontje/received_events",
         "repos_url" : "https://api.github.com/users/jorisbontje/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/jorisbontje/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/jorisbontje/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/jorisbontje"
      }
   },
   {
      "body" : "Silent corruption in raid arrays is absolutely possible. Its possible this is an issue with leveldb, but it seems more likely there are silent errors somewhere in your disk/raid controller/etc.",
      "created_at" : "2015-08-13T10:33:55Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-130606728",
      "id" : 130606728,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-08-13T10:33:55Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/130606728",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/649246?v=3",
         "events_url" : "https://api.github.com/users/TheBlueMatt/events{/privacy}",
         "followers_url" : "https://api.github.com/users/TheBlueMatt/followers",
         "following_url" : "https://api.github.com/users/TheBlueMatt/following{/other_user}",
         "gists_url" : "https://api.github.com/users/TheBlueMatt/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/TheBlueMatt",
         "id" : 649246,
         "login" : "TheBlueMatt",
         "organizations_url" : "https://api.github.com/users/TheBlueMatt/orgs",
         "received_events_url" : "https://api.github.com/users/TheBlueMatt/received_events",
         "repos_url" : "https://api.github.com/users/TheBlueMatt/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/TheBlueMatt/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/TheBlueMatt/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/TheBlueMatt"
      }
   },
   {
      "body" : "Have seen something similar in dogecoin/dogecoin#1216 - could be specific to our changes, but the version in question is a complete rebuild around Bitcoin Core 0.11. Either way, am looking into this, if we find anything we will of course push fixes into Bitcoin as well.\r\n",
      "created_at" : "2015-08-13T12:03:34Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-130643595",
      "id" : 130643595,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-08-13T12:03:34Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/130643595",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/3513077?v=3",
         "events_url" : "https://api.github.com/users/rnicoll/events{/privacy}",
         "followers_url" : "https://api.github.com/users/rnicoll/followers",
         "following_url" : "https://api.github.com/users/rnicoll/following{/other_user}",
         "gists_url" : "https://api.github.com/users/rnicoll/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/rnicoll",
         "id" : 3513077,
         "login" : "rnicoll",
         "organizations_url" : "https://api.github.com/users/rnicoll/orgs",
         "received_events_url" : "https://api.github.com/users/rnicoll/received_events",
         "repos_url" : "https://api.github.com/users/rnicoll/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/rnicoll/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/rnicoll/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/rnicoll"
      }
   },
   {
      "body" : "@jorisbontje Is it possible that you ran out of memory or the disk was full?",
      "created_at" : "2015-08-13T12:40:43Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-130655769",
      "id" : 130655769,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-08-13T12:40:43Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/130655769",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/548488?v=3",
         "events_url" : "https://api.github.com/users/sipa/events{/privacy}",
         "followers_url" : "https://api.github.com/users/sipa/followers",
         "following_url" : "https://api.github.com/users/sipa/following{/other_user}",
         "gists_url" : "https://api.github.com/users/sipa/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/sipa",
         "id" : 548488,
         "login" : "sipa",
         "organizations_url" : "https://api.github.com/users/sipa/orgs",
         "received_events_url" : "https://api.github.com/users/sipa/received_events",
         "repos_url" : "https://api.github.com/users/sipa/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/sipa/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/sipa/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/sipa"
      }
   },
   {
      "body" : "@sipa enough disk space (50GB+). I was expecting an out of memory (only 3GB total), but didn't get an Out of memory error anywhere. I configured the relay settings to reduce the pending tx pool, but still happening.\r\n\r\nI can try to reproduce this with a `-reindex`, is there any kind of additional monitoring / debug logging, etc you'd like me to enable?",
      "created_at" : "2015-08-13T13:39:41Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-130675638",
      "id" : 130675638,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-08-13T13:39:41Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/130675638",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/620959?v=3",
         "events_url" : "https://api.github.com/users/jorisbontje/events{/privacy}",
         "followers_url" : "https://api.github.com/users/jorisbontje/followers",
         "following_url" : "https://api.github.com/users/jorisbontje/following{/other_user}",
         "gists_url" : "https://api.github.com/users/jorisbontje/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/jorisbontje",
         "id" : 620959,
         "login" : "jorisbontje",
         "organizations_url" : "https://api.github.com/users/jorisbontje/orgs",
         "received_events_url" : "https://api.github.com/users/jorisbontje/received_events",
         "repos_url" : "https://api.github.com/users/jorisbontje/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/jorisbontje/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/jorisbontje/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/jorisbontje"
      }
   },
   {
      "body" : "Joris: does the crash happen while you are synchronizing/running, or after\na stop/start?\n\nIf so, could you try running with -par=1 (to reduce risk for CPU problems,\nbut I don't think that is the problem here, you'd get validation errors\ninstead).\n\nMy last guess would be an incompatibility between LevelDB and the\nfilesystem yo're using.\n",
      "created_at" : "2015-08-13T15:06:07Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-130723317",
      "id" : 130723317,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-08-13T15:06:07Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/130723317",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/548488?v=3",
         "events_url" : "https://api.github.com/users/sipa/events{/privacy}",
         "followers_url" : "https://api.github.com/users/sipa/followers",
         "following_url" : "https://api.github.com/users/sipa/following{/other_user}",
         "gists_url" : "https://api.github.com/users/sipa/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/sipa",
         "id" : 548488,
         "login" : "sipa",
         "organizations_url" : "https://api.github.com/users/sipa/orgs",
         "received_events_url" : "https://api.github.com/users/sipa/received_events",
         "repos_url" : "https://api.github.com/users/sipa/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/sipa/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/sipa/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/sipa"
      }
   },
   {
      "body" : "This happened again while synchronizing/running `bitcoind -reindex -par=1`\r\n\r\n```\r\n2015-08-20 16:56:48 UpdateTip: new best=0000000000000000043a35f28c76f951eef172b33a53a49c47a891c759943341  height=370730  log2_work=83.236584  tx=80585198  date=2015-08-20 16:55:41 progress=0.999999  cache=62.6MiB(15705tx)\r\n2015-08-20 16:56:48 UpdateTip: 1 of last 100 blocks above version 3\r\n2015-08-20 16:56:48 keypool reserve 3695\r\n2015-08-20 16:56:48 keypool return 3695\r\n2015-08-20 16:56:49 Corruption: block checksum mismatch\r\n2015-08-20 16:56:49 *** System error while flushing: Database corrupted\r\n2015-08-20 16:56:49 Error: Error: A fatal internal error occured, see debug.log for details\r\n2015-08-20 16:56:49 ERROR: ProcessNewBlock: ActivateBestChain failed\r\n2015-08-20 16:56:49 net thread interrupt\r\n2015-08-20 16:56:49 opencon thread interrupt\r\n2015-08-20 16:56:49 addcon thread interrupt\r\n2015-08-20 16:56:49 scheduler thread interrupt\r\n2015-08-20 16:56:49 msghand thread interrupt\r\n2015-08-20 16:56:49 Shutdown: In progress...\r\n2015-08-20 16:56:49 StopNode()\r\n2015-08-20 16:56:50 Corruption: block checksum mismatch\r\n2015-08-20 16:56:50 *** System error while flushing: Database corrupted\r\n2015-08-20 16:56:50 Error: Error: A fatal internal error occured, see debug.log for details\r\n2015-08-20 16:56:51 Shutdown: done\r\n```",
      "created_at" : "2015-08-20T19:03:25Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-133121796",
      "id" : 133121796,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-08-20T19:03:25Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/133121796",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/620959?v=3",
         "events_url" : "https://api.github.com/users/jorisbontje/events{/privacy}",
         "followers_url" : "https://api.github.com/users/jorisbontje/followers",
         "following_url" : "https://api.github.com/users/jorisbontje/following{/other_user}",
         "gists_url" : "https://api.github.com/users/jorisbontje/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/jorisbontje",
         "id" : 620959,
         "login" : "jorisbontje",
         "organizations_url" : "https://api.github.com/users/jorisbontje/orgs",
         "received_events_url" : "https://api.github.com/users/jorisbontje/received_events",
         "repos_url" : "https://api.github.com/users/jorisbontje/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/jorisbontje/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/jorisbontje/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/jorisbontje"
      }
   },
   {
      "body" : "I'm experiencing similar problem. Ubuntu 12.04.5 on a physical box, software raid.",
      "created_at" : "2015-08-25T10:32:36Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-134549506",
      "id" : 134549506,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-08-25T10:32:36Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/134549506",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/2736855?v=3",
         "events_url" : "https://api.github.com/users/Mikadily/events{/privacy}",
         "followers_url" : "https://api.github.com/users/Mikadily/followers",
         "following_url" : "https://api.github.com/users/Mikadily/following{/other_user}",
         "gists_url" : "https://api.github.com/users/Mikadily/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/Mikadily",
         "id" : 2736855,
         "login" : "Mikadily",
         "organizations_url" : "https://api.github.com/users/Mikadily/orgs",
         "received_events_url" : "https://api.github.com/users/Mikadily/received_events",
         "repos_url" : "https://api.github.com/users/Mikadily/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/Mikadily/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/Mikadily/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/Mikadily"
      }
   },
   {
      "body" : "I have the same problem here. It all started with the following message \"ERROR: AcceptToMemoryPool: non-final\". Then after stopping bitcoind I get this message \r\n\"Corruption: block checksum mismatch\r\n*** System error while flushing: Database corrupted\"",
      "created_at" : "2015-09-10T13:55:48Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-139241582",
      "id" : 139241582,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-09-14T15:19:36Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/139241582",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/9290908?v=3",
         "events_url" : "https://api.github.com/users/miguelmorales85/events{/privacy}",
         "followers_url" : "https://api.github.com/users/miguelmorales85/followers",
         "following_url" : "https://api.github.com/users/miguelmorales85/following{/other_user}",
         "gists_url" : "https://api.github.com/users/miguelmorales85/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/miguelmorales85",
         "id" : 9290908,
         "login" : "miguelmorales85",
         "organizations_url" : "https://api.github.com/users/miguelmorales85/orgs",
         "received_events_url" : "https://api.github.com/users/miguelmorales85/received_events",
         "repos_url" : "https://api.github.com/users/miguelmorales85/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/miguelmorales85/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/miguelmorales85/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/miguelmorales85"
      }
   },
   {
      "body" : "@miguelmorales85 what kind of filesystem / storage setup do you have? (raid, vps, simfs, etc)",
      "created_at" : "2015-09-10T14:00:10Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-139242696",
      "id" : 139242696,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-09-10T14:00:10Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/139242696",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/620959?v=3",
         "events_url" : "https://api.github.com/users/jorisbontje/events{/privacy}",
         "followers_url" : "https://api.github.com/users/jorisbontje/followers",
         "following_url" : "https://api.github.com/users/jorisbontje/following{/other_user}",
         "gists_url" : "https://api.github.com/users/jorisbontje/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/jorisbontje",
         "id" : 620959,
         "login" : "jorisbontje",
         "organizations_url" : "https://api.github.com/users/jorisbontje/orgs",
         "received_events_url" : "https://api.github.com/users/jorisbontje/received_events",
         "repos_url" : "https://api.github.com/users/jorisbontje/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/jorisbontje/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/jorisbontje/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/jorisbontje"
      }
   },
   {
      "body" : "@jorisbontje the hd was formated in ext4",
      "created_at" : "2015-09-10T14:06:45Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-139245262",
      "id" : 139245262,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-09-10T14:06:45Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/139245262",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/9290908?v=3",
         "events_url" : "https://api.github.com/users/miguelmorales85/events{/privacy}",
         "followers_url" : "https://api.github.com/users/miguelmorales85/followers",
         "following_url" : "https://api.github.com/users/miguelmorales85/following{/other_user}",
         "gists_url" : "https://api.github.com/users/miguelmorales85/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/miguelmorales85",
         "id" : 9290908,
         "login" : "miguelmorales85",
         "organizations_url" : "https://api.github.com/users/miguelmorales85/orgs",
         "received_events_url" : "https://api.github.com/users/miguelmorales85/received_events",
         "repos_url" : "https://api.github.com/users/miguelmorales85/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/miguelmorales85/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/miguelmorales85/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/miguelmorales85"
      }
   },
   {
      "body" : "I have the same issue. and I don't have disk corruption.\r\nMy database was newly imported from bootstrap.dat\r\n\r\nThis is Bitcoin Core 0.11 (64-bit), and on Windows 2008 R2 (64-bit), NTFS. I have about 1TB free.\r\n\r\n2015-09-14 08:42:33 UpdateTip: new best=0000000000000000184d37ea91dc37ad61f2c9eca556020476787bf9916a5a24  height=337433  log2_work=81.923139  tx=55737117  date=2015-01-04 11:54:53 progress=0.635931  cache=62.6MiB(10952tx)\r\n2015-09-14 08:42:34 Corruption: block checksum mismatch\r\n2015-09-14 08:42:34 *** System error while flushing: Database corrupted\r\n2015-09-14 08:44:13 ERROR: ProcessNewBlock: ActivateBestChain failed\r\n2015-09-14 08:44:13 opencon thread interrupt\r\n2015-09-14 08:44:13 scheduler thread interrupt\r\n2015-09-14 08:44:13 addcon thread interrupt\r\n2015-09-14 08:44:13 net thread interrupt\r\n2015-09-14 08:44:14 ERROR: AcceptToMemoryPool: non-final\r\n2015-09-14 08:44:14 msghand thread interrupt\r\n2015-09-14 08:44:14 Shutdown: In progress...\r\n2015-09-14 08:44:14 StopNode()\r\n2015-09-14 08:44:14 UPNP_DeletePortMapping() returned: 0\r\n2015-09-14 08:44:14 upnp thread interrupt\r\n2015-09-14 08:44:14 Corruption: block checksum mismatch\r\n2015-09-14 08:44:14 *** System error while flushing: Database corrupted\r\n2015-09-14 08:44:16 Shutdown: done\r\n",
      "created_at" : "2015-09-14T08:48:24Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-140003877",
      "id" : 140003877,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-09-14T08:49:56Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/140003877",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/1757930?v=3",
         "events_url" : "https://api.github.com/users/chuacw/events{/privacy}",
         "followers_url" : "https://api.github.com/users/chuacw/followers",
         "following_url" : "https://api.github.com/users/chuacw/following{/other_user}",
         "gists_url" : "https://api.github.com/users/chuacw/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/chuacw",
         "id" : 1757930,
         "login" : "chuacw",
         "organizations_url" : "https://api.github.com/users/chuacw/orgs",
         "received_events_url" : "https://api.github.com/users/chuacw/received_events",
         "repos_url" : "https://api.github.com/users/chuacw/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/chuacw/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/chuacw/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/chuacw"
      }
   },
   {
      "body" : "Again database corruption at block 252.dat\r\nI am reindexing.. this takes a while so I will let you know the results...",
      "created_at" : "2015-09-14T15:20:20Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-140114026",
      "id" : 140114026,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-09-14T15:20:20Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/140114026",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/9290908?v=3",
         "events_url" : "https://api.github.com/users/miguelmorales85/events{/privacy}",
         "followers_url" : "https://api.github.com/users/miguelmorales85/followers",
         "following_url" : "https://api.github.com/users/miguelmorales85/following{/other_user}",
         "gists_url" : "https://api.github.com/users/miguelmorales85/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/miguelmorales85",
         "id" : 9290908,
         "login" : "miguelmorales85",
         "organizations_url" : "https://api.github.com/users/miguelmorales85/orgs",
         "received_events_url" : "https://api.github.com/users/miguelmorales85/received_events",
         "repos_url" : "https://api.github.com/users/miguelmorales85/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/miguelmorales85/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/miguelmorales85/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/miguelmorales85"
      }
   },
   {
      "body" : "I have just failed to reindex bitcoind.. this is what happened. \r\n2015-09-16 05:15:09 UpdateTip: new best=0000000000000000087d41de97e50f2c63151b0caee4a63f1387dd0eb8ae2153  height=345409  log2_work=82.326044  tx=60898518  date=2015-02-27 14:50:06 progress=0.724568  cache=64.9MiB(14575tx)\r\n2015-09-16 05:15:09 Corruption: block checksum mismatch\r\n2015-09-16 05:15:09 *** System error while flushing: Database corrupted\r\n2015-09-16 05:15:10 Error: Error: A fatal internal error occured, see debug.log for details\r\n2015-09-16 05:15:10 ERROR: ProcessNewBlock: ActivateBestChain failed\r\n2015-09-16 05:15:10 addcon thread interrupt\r\n2015-09-16 05:15:10 opencon thread interrupt\r\n2015-09-16 05:15:10 scheduler thread interrupt\r\n2015-09-16 05:15:10 net thread interrupt\r\n2015-09-16 05:15:10 msghand thread interrupt\r\n2015-09-16 05:15:10 Shutdown: In progress...\r\n2015-09-16 05:15:10 RPCAcceptHandler: Error: Operation canceled\r\n2015-09-16 05:15:10 StopNode()\r\n2015-09-16 05:15:11 Corruption: block checksum mismatch\r\n2015-09-16 05:15:11 *** System error while flushing: Database corrupted\r\n2015-09-16 05:15:11 Error: Error: A fatal internal error occured, see debug.log for details\r\n2015-09-16 05:15:11 Shutdown: done\r\n\r\ncan somebody help me. Please dont tell me I have to reindex AGAIN... ",
      "created_at" : "2015-09-16T13:43:08Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-140746045",
      "id" : 140746045,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-09-16T13:43:08Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/140746045",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/9290908?v=3",
         "events_url" : "https://api.github.com/users/miguelmorales85/events{/privacy}",
         "followers_url" : "https://api.github.com/users/miguelmorales85/followers",
         "following_url" : "https://api.github.com/users/miguelmorales85/following{/other_user}",
         "gists_url" : "https://api.github.com/users/miguelmorales85/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/miguelmorales85",
         "id" : 9290908,
         "login" : "miguelmorales85",
         "organizations_url" : "https://api.github.com/users/miguelmorales85/orgs",
         "received_events_url" : "https://api.github.com/users/miguelmorales85/received_events",
         "repos_url" : "https://api.github.com/users/miguelmorales85/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/miguelmorales85/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/miguelmorales85/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/miguelmorales85"
      }
   },
   {
      "body" : "@miguelmorales85 it is highly likely you have hardware problems",
      "created_at" : "2015-09-16T13:53:23Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-140748547",
      "id" : 140748547,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-09-16T13:53:23Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/140748547",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/494411?v=3",
         "events_url" : "https://api.github.com/users/jgarzik/events{/privacy}",
         "followers_url" : "https://api.github.com/users/jgarzik/followers",
         "following_url" : "https://api.github.com/users/jgarzik/following{/other_user}",
         "gists_url" : "https://api.github.com/users/jgarzik/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/jgarzik",
         "id" : 494411,
         "login" : "jgarzik",
         "organizations_url" : "https://api.github.com/users/jgarzik/orgs",
         "received_events_url" : "https://api.github.com/users/jgarzik/received_events",
         "repos_url" : "https://api.github.com/users/jgarzik/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/jgarzik/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/jgarzik/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/jgarzik"
      }
   },
   {
      "body" : "@jgarzik I ran e2fsk on the unmounted partition in a live session with 0 errors :(",
      "created_at" : "2015-09-16T15:42:49Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-140781631",
      "id" : 140781631,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-09-16T15:42:49Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/140781631",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/9290908?v=3",
         "events_url" : "https://api.github.com/users/miguelmorales85/events{/privacy}",
         "followers_url" : "https://api.github.com/users/miguelmorales85/followers",
         "following_url" : "https://api.github.com/users/miguelmorales85/following{/other_user}",
         "gists_url" : "https://api.github.com/users/miguelmorales85/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/miguelmorales85",
         "id" : 9290908,
         "login" : "miguelmorales85",
         "organizations_url" : "https://api.github.com/users/miguelmorales85/orgs",
         "received_events_url" : "https://api.github.com/users/miguelmorales85/received_events",
         "repos_url" : "https://api.github.com/users/miguelmorales85/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/miguelmorales85/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/miguelmorales85/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/miguelmorales85"
      }
   },
   {
      "body" : "Firstly, I created my account because of this issue.\r\nMaybe this isn't a hardware issue, as I am also running x64 Linux (ubuntu 14.04) w/ no disk errors.\r\nI've tried reindexing around 6 times since thursday. 5 out of those times I hit database corruption at random points, but before this last time it hit I had filled my ext4 drive when it hit 2 weeks til completion.\r\nSo I set up a symlink so the blocks directory is stored on a much larger ntfs drive. Go to reindex on next launch because it's required and I hit database corruption at 52 weeks. I literally just purchased the ntfs drive a month ago, and I've not had any errors reported on either disks.",
      "created_at" : "2015-09-27T14:21:57Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-143561275",
      "id" : 143561275,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-09-27T14:21:57Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/143561275",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/14860307?v=3",
         "events_url" : "https://api.github.com/users/artisbigshirts/events{/privacy}",
         "followers_url" : "https://api.github.com/users/artisbigshirts/followers",
         "following_url" : "https://api.github.com/users/artisbigshirts/following{/other_user}",
         "gists_url" : "https://api.github.com/users/artisbigshirts/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/artisbigshirts",
         "id" : 14860307,
         "login" : "artisbigshirts",
         "organizations_url" : "https://api.github.com/users/artisbigshirts/orgs",
         "received_events_url" : "https://api.github.com/users/artisbigshirts/received_events",
         "repos_url" : "https://api.github.com/users/artisbigshirts/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/artisbigshirts/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/artisbigshirts/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/artisbigshirts"
      }
   },
   {
      "body" : "I finally finally did not get the checksum mismatch after several reindexing (like a week of try and error). But now the node keeps getting STALLED, I have can not SYNC the node. The closer I got was like verification progress to 0.9989xxxx and I am slowly going down... Can somebody help me? I have restarted the node, reboot the Ubuntu PC and I just get the node UP and then STALLED. Of course I am accepting conections.",
      "created_at" : "2015-09-27T14:47:24Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-143563729",
      "id" : 143563729,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-09-27T14:47:24Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/143563729",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/9290908?v=3",
         "events_url" : "https://api.github.com/users/miguelmorales85/events{/privacy}",
         "followers_url" : "https://api.github.com/users/miguelmorales85/followers",
         "following_url" : "https://api.github.com/users/miguelmorales85/following{/other_user}",
         "gists_url" : "https://api.github.com/users/miguelmorales85/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/miguelmorales85",
         "id" : 9290908,
         "login" : "miguelmorales85",
         "organizations_url" : "https://api.github.com/users/miguelmorales85/orgs",
         "received_events_url" : "https://api.github.com/users/miguelmorales85/received_events",
         "repos_url" : "https://api.github.com/users/miguelmorales85/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/miguelmorales85/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/miguelmorales85/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/miguelmorales85"
      }
   },
   {
      "body" : "Miguel: anything in debug.log mentioning invalid blocks?\n",
      "created_at" : "2015-09-27T14:52:48Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-143563960",
      "id" : 143563960,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-09-27T14:52:48Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/143563960",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/548488?v=3",
         "events_url" : "https://api.github.com/users/sipa/events{/privacy}",
         "followers_url" : "https://api.github.com/users/sipa/followers",
         "following_url" : "https://api.github.com/users/sipa/following{/other_user}",
         "gists_url" : "https://api.github.com/users/sipa/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/sipa",
         "id" : 548488,
         "login" : "sipa",
         "organizations_url" : "https://api.github.com/users/sipa/orgs",
         "received_events_url" : "https://api.github.com/users/sipa/received_events",
         "repos_url" : "https://api.github.com/users/sipa/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/sipa/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/sipa/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/sipa"
      }
   },
   {
      "body" : "@sipa  Hello, I have run  \"cat /home/USER/.bitcoin/debug.log | grep invalid\"\r\nand found this: \r\n2015-09-26 02:50:28 ERROR: AcceptBlockHeader: block is marked invalid\r\n2015-09-26 02:50:28 ERROR: invalid header received\r\n2015-09-26 04:49:47 ERROR: AcceptBlockHeader: block is marked invalid\r\n2015-09-26 04:49:47 ERROR: invalid header received\r\n2015-09-26 07:25:55 ERROR: AcceptBlockHeader: block is marked invalid\r\n2015-09-26 07:25:55 ERROR: invalid header received\r\n\r\nwhat can I do about it?",
      "created_at" : "2015-09-27T14:56:43Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-143564132",
      "id" : 143564132,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-09-27T14:56:43Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/143564132",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/9290908?v=3",
         "events_url" : "https://api.github.com/users/miguelmorales85/events{/privacy}",
         "followers_url" : "https://api.github.com/users/miguelmorales85/followers",
         "following_url" : "https://api.github.com/users/miguelmorales85/following{/other_user}",
         "gists_url" : "https://api.github.com/users/miguelmorales85/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/miguelmorales85",
         "id" : 9290908,
         "login" : "miguelmorales85",
         "organizations_url" : "https://api.github.com/users/miguelmorales85/orgs",
         "received_events_url" : "https://api.github.com/users/miguelmorales85/received_events",
         "repos_url" : "https://api.github.com/users/miguelmorales85/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/miguelmorales85/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/miguelmorales85/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/miguelmorales85"
      }
   },
   {
      "body" : "That almost certainly means Bitcoin Core incorrectly decided that a\nsignature or transaction was invalid in a block, and subsequently marked\nthe block invalid.\n\nCan you try reindexing with -par=1 or par=1 in bitcoin.conf?\n",
      "created_at" : "2015-09-27T14:59:53Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-143564881",
      "id" : 143564881,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-09-27T14:59:53Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/143564881",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/548488?v=3",
         "events_url" : "https://api.github.com/users/sipa/events{/privacy}",
         "followers_url" : "https://api.github.com/users/sipa/followers",
         "following_url" : "https://api.github.com/users/sipa/following{/other_user}",
         "gists_url" : "https://api.github.com/users/sipa/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/sipa",
         "id" : 548488,
         "login" : "sipa",
         "organizations_url" : "https://api.github.com/users/sipa/orgs",
         "received_events_url" : "https://api.github.com/users/sipa/received_events",
         "repos_url" : "https://api.github.com/users/sipa/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/sipa/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/sipa/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/sipa"
      }
   },
   {
      "body" : "oh god... I feared you would say that... Ok .. will do. See you in a few days. Thanks.",
      "created_at" : "2015-09-27T15:04:26Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-143565841",
      "id" : 143565841,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-09-27T15:04:26Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/143565841",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/9290908?v=3",
         "events_url" : "https://api.github.com/users/miguelmorales85/events{/privacy}",
         "followers_url" : "https://api.github.com/users/miguelmorales85/followers",
         "following_url" : "https://api.github.com/users/miguelmorales85/following{/other_user}",
         "gists_url" : "https://api.github.com/users/miguelmorales85/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/miguelmorales85",
         "id" : 9290908,
         "login" : "miguelmorales85",
         "organizations_url" : "https://api.github.com/users/miguelmorales85/orgs",
         "received_events_url" : "https://api.github.com/users/miguelmorales85/received_events",
         "repos_url" : "https://api.github.com/users/miguelmorales85/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/miguelmorales85/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/miguelmorales85/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/miguelmorales85"
      }
   },
   {
      "body" : "Hello @sipa  I have reindexed. I have the same problem, I can never get to SYNC the node. At this moment the node progression is at 0.9983xxxx and going down and down every seccond... it's going to Stall again. So no par=1 in bitcoin.conf did something to prevent my problem. ",
      "created_at" : "2015-10-07T08:55:05Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-146122105",
      "id" : 146122105,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-10-07T08:55:05Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/146122105",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/9290908?v=3",
         "events_url" : "https://api.github.com/users/miguelmorales85/events{/privacy}",
         "followers_url" : "https://api.github.com/users/miguelmorales85/followers",
         "following_url" : "https://api.github.com/users/miguelmorales85/following{/other_user}",
         "gists_url" : "https://api.github.com/users/miguelmorales85/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/miguelmorales85",
         "id" : 9290908,
         "login" : "miguelmorales85",
         "organizations_url" : "https://api.github.com/users/miguelmorales85/orgs",
         "received_events_url" : "https://api.github.com/users/miguelmorales85/received_events",
         "repos_url" : "https://api.github.com/users/miguelmorales85/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/miguelmorales85/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/miguelmorales85/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/miguelmorales85"
      }
   },
   {
      "body" : "Hi,\r\nI have got this several times. About 1-2 times per week. When I started bitcoind again, it working ok and I don't need to do reindex it again. So this is not so fatal, but annoying.\r\nMy Bitcoin core version is quite new. I compiled it from source (master) about 2-3 weeks ago. Now there are over 200 commits more than this version.\r\nI think pure 0.11.0 version working better.\r\n\r\nHere is the last log on yestersay:\r\n2015-10-07 11:29:45 receive version message: /bitcoinseeder:0.01/: version 60000, blocks=230000, us=88.193.138.220:8333, peer=593\r\n2015-10-07 11:32:39 receive version message: /Snoopy:0.2.1/: version 70001, blocks=0, us=88.193.138.220:8333, peer=594\r\n2015-10-07 11:32:42 receive version message: /bitcoinj:0.14SNAPSHOT/: version 70001, blocks=377853, us=127.0.0.1:8333, peer=595\r\n2015-10-07 11:33:20 Corruption: block checksum mismatch\r\n2015-10-07 11:33:20 *** System error while flushing: Database corrupted\r\n2015-10-07 11:33:20 Error: Error: A fatal internal error occurred, see debug.log for details\r\n2015-10-07 11:33:20 ERROR: ProcessNewBlock: ActivateBestChain failed\r\n2015-10-07 11:33:21 scheduler thread interrupt\r\n2015-10-07 11:33:21 opencon thread interrupt\r\n2015-10-07 11:33:21 addcon thread interrupt\r\n2015-10-07 11:33:21 msghand thread interrupt\r\n2015-10-07 11:33:21 net thread interrupt\r\n2015-10-07 11:33:21 Shutdown: In progress...\r\n2015-10-07 11:33:21 StopNode()\r\n2015-10-07 11:33:21 Corruption: block checksum mismatch\r\n2015-10-07 11:33:21 *** System error while flushing: Database corrupted\r\n2015-10-07 11:33:21 Error: Error: A fatal internal error occurred, see debug.log for details\r\n2015-10-07 11:33:21 Shutdown: done\r\n\r\nWhen this occurred next time, I'll pull new sources from master branch and try again.\r\n",
      "created_at" : "2015-10-08T04:44:11Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-146418655",
      "id" : 146418655,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-10-08T05:49:57Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/146418655",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/8566191?v=3",
         "events_url" : "https://api.github.com/users/santzi/events{/privacy}",
         "followers_url" : "https://api.github.com/users/santzi/followers",
         "following_url" : "https://api.github.com/users/santzi/following{/other_user}",
         "gists_url" : "https://api.github.com/users/santzi/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/santzi",
         "id" : 8566191,
         "login" : "santzi",
         "organizations_url" : "https://api.github.com/users/santzi/orgs",
         "received_events_url" : "https://api.github.com/users/santzi/received_events",
         "repos_url" : "https://api.github.com/users/santzi/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/santzi/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/santzi/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/santzi"
      }
   },
   {
      "body" : "With bitcoind 0.11.1 on a 64-bit Linux PC I got the \"Corruption: block checksum mismatch\" error.\r\nI ran a full system check, which reported single bit data dependent corruption in one of the DIMMs.\r\nI think the LevelDB data was corrupted when cached in the DRAM memory and then got written\r\nback to disk. I replaced both DIMMs, ran with -reindex=1, and bitcoind has been running without\r\nerror for a week.\r\n",
      "created_at" : "2015-10-28T20:02:29Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-151973844",
      "id" : 151973844,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-10-28T20:02:29Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/151973844",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/8991626?v=3",
         "events_url" : "https://api.github.com/users/trevelocity/events{/privacy}",
         "followers_url" : "https://api.github.com/users/trevelocity/followers",
         "following_url" : "https://api.github.com/users/trevelocity/following{/other_user}",
         "gists_url" : "https://api.github.com/users/trevelocity/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/trevelocity",
         "id" : 8991626,
         "login" : "trevelocity",
         "organizations_url" : "https://api.github.com/users/trevelocity/orgs",
         "received_events_url" : "https://api.github.com/users/trevelocity/received_events",
         "repos_url" : "https://api.github.com/users/trevelocity/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/trevelocity/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/trevelocity/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/trevelocity"
      }
   },
   {
      "body" : "Have the same on Ubuntu 15.04 x64 + Bitcoin core 0.11.1.0 (VM 2GB ram, enough hard drive space, ram check OK). First it happened 2 days ago (24hr client just receiving info most of the time), I deleted last 3 blk filed, reindex started, on -2y30w throws \"error reading from database\". Yesterday I cleared btc data directory except bitcoin.conf and wallet.dat files, moved btc directory to different physical drive with zfs, after resync from scratch it throws \"error reading from database\" on -2y30w again. \r\n\r\n```\r\n2015-12-17 00:35:19 UpdateTip: new best=000000000000005bb239fc8ff69c2d9fd33d1d23532dcebcb24192758f1baba3  height=237055  log2_work=70.080825  tx=18133567  date=2013-05-20 12:51:09 progress=0.081912  cache=4.8MiB(3525tx)\r\n2015-12-17 00:35:19 UpdateTip: new best=000000000000015767a84949b0bbf7ba25c24515e6198fd57909b8d3d1d7a7f5  height=237056  log2_work=70.08088  tx=18133568  date=2013-05-20 12:51:48 progress=0.081912  cache=4.8MiB(3526tx)\r\n2015-12-17 00:35:19 UpdateTip: new best=0000000000000156d33717e7055beb017cd07e0d044fc7a4a364cf459d148aa1  height=237057  log2_work=70.080936  tx=18134049  date=2013-05-20 12:55:01 progress=0.081914  cache=5.0MiB(4225tx)\r\n2015-12-17 00:35:20 UpdateTip: new best=00000000000000ce70375942e44e5a5f5f2192afecdc178fe0fe240bb8310d1c  height=237058  log2_work=70.080991  tx=18134844  date=2013-05-20 13:18:02 progress=0.081917  cache=5.9MiB(5339tx)\r\n2015-12-17 00:35:20 UpdateTip: new best=00000000000001679e03844bc5e651ff9394b57f4612a43e555fc5cedab5c3b2  height=237059  log2_work=70.081047  tx=18135625  date=2013-05-20 13:51:19 progress=0.081921  cache=6.2MiB(6234tx)\r\n2015-12-17 00:35:20 UpdateTip: new best=000000000000001a74e886007d45abf8feae6668516e35462fe207f0bfe48b7e  height=237060  log2_work=70.081102  tx=18135989  date=2013-05-20 14:02:32 progress=0.081923  cache=6.9MiB(7048tx)\r\n2015-12-17 00:35:20 UpdateTip: new best=0000000000000027162a00ff456c8f1f2a37ac915714c17b9d2c622dab9b94e6  height=237061  log2_work=70.081158  tx=18136400  date=2013-05-20 14:03:09 progress=0.081924  cache=7.0MiB(7490tx)\r\n2015-12-17 00:35:20 UpdateTip: new best=00000000000000f6f3787321ff929f44a9bfc0e75ac54ac90157e5bf5dd23cb6  height=237062  log2_work=70.081213  tx=18136619  date=2013-05-20 14:14:18 progress=0.081925  cache=7.4MiB(7899tx)\r\n2015-12-17 00:35:20 UpdateTip: new best=0000000000000158fc5cee0bcac58973ddc158e4891df80eef7c67f78e5cd47e  height=237063  log2_work=70.081269  tx=18136747  date=2013-05-20 14:17:25 progress=0.081926  cache=7.8MiB(8135tx)\r\n2015-12-17 00:35:20 LevelDB read failure: Corruption: block checksum mismatch\r\n2015-12-17 00:35:20 Corruption: block checksum mismatch\r\n```\r\n\r\n```\r\n2015-12-17 09:36:10 init message: Rescanning...\r\n2015-12-17 09:36:10 Rescanning last 1101 blocks (from block 235950)...\r\n2015-12-17 09:36:10  rescan                    1ms\r\n2015-12-17 09:36:10 init message: Activating best chain...\r\n2015-12-17 09:36:11 UpdateTip: new best=000000000000008bdabb35a3ce5f85b8c3a11ffefd5a29defcd4c786e271f8a9  height=237052  log2_work=70.080658  tx=18132299  date=2013-05-20 11:49:50 progress=0.081864  cache=1.2MiB(809tx)\r\n2015-12-17 09:36:11 UpdateTip: new best=00000000000000b9f7b4bfaee1287b3e51fd534cdb30f42c482dfff7f714f6d3  height=237053  log2_work=70.080714  tx=18132304  date=2013-05-20 12:10:02 progress=0.081864  cache=1.3MiB(820tx)\r\n2015-12-17 09:36:11 UpdateTip: new best=00000000000000afe58811aea548dac4d6f4fd8c9c2b88836c84170ab52b03b8  height=237054  log2_work=70.080769  tx=18133326  date=2013-05-20 12:35:09 progress=0.081869  cache=3.6MiB(3003tx)\r\n2015-12-17 09:36:12 UpdateTip: new best=000000000000005bb239fc8ff69c2d9fd33d1d23532dcebcb24192758f1baba3  height=237055  log2_work=70.080825  tx=18133567  date=2013-05-20 12:51:09 progress=0.081870  cache=4.3MiB(3525tx)\r\n2015-12-17 09:36:12 UpdateTip: new best=000000000000015767a84949b0bbf7ba25c24515e6198fd57909b8d3d1d7a7f5  height=237056  log2_work=70.08088  tx=18133568  date=2013-05-20 12:51:48 progress=0.081870  cache=4.3MiB(3526tx)\r\n2015-12-17 09:36:12 UpdateTip: new best=0000000000000156d33717e7055beb017cd07e0d044fc7a4a364cf459d148aa1  height=237057  log2_work=70.080936  tx=18134049  date=2013-05-20 12:55:01 progress=0.081872  cache=4.5MiB(4225tx)\r\n2015-12-17 09:36:12 UpdateTip: new best=00000000000000ce70375942e44e5a5f5f2192afecdc178fe0fe240bb8310d1c  height=237058  log2_work=70.080991  tx=18134844  date=2013-05-20 13:18:02 progress=0.081876  cache=5.4MiB(5339tx)\r\n2015-12-17 09:36:12 UpdateTip: new best=00000000000001679e03844bc5e651ff9394b57f4612a43e555fc5cedab5c3b2  height=237059  log2_work=70.081047  tx=18135625  date=2013-05-20 13:51:19 progress=0.081879  cache=5.8MiB(6234tx)\r\n2015-12-17 09:36:13 UpdateTip: new best=000000000000001a74e886007d45abf8feae6668516e35462fe207f0bfe48b7e  height=237060  log2_work=70.081102  tx=18135989  date=2013-05-20 14:02:32 progress=0.081881  cache=6.5MiB(7048tx)\r\n2015-12-17 09:36:13 UpdateTip: new best=0000000000000027162a00ff456c8f1f2a37ac915714c17b9d2c622dab9b94e6  height=237061  log2_work=70.081158  tx=18136400  date=2013-05-20 14:03:09 progress=0.081883  cache=6.6MiB(7490tx)\r\n2015-12-17 09:36:13 UpdateTip: new best=00000000000000f6f3787321ff929f44a9bfc0e75ac54ac90157e5bf5dd23cb6  height=237062  log2_work=70.081213  tx=18136619  date=2013-05-20 14:14:18 progress=0.081884  cache=7.0MiB(7899tx)\r\n2015-12-17 09:36:13 UpdateTip: new best=0000000000000158fc5cee0bcac58973ddc158e4891df80eef7c67f78e5cd47e  height=237063  log2_work=70.081269  tx=18136747  date=2013-05-20 14:17:25 progress=0.081884  cache=7.3MiB(8135tx)\r\n2015-12-17 09:36:13 LevelDB read failure: Corruption: block checksum mismatch\r\n2015-12-17 09:36:13 Corruption: block checksum mismatch\r\n2015-12-17 09:36:53 Error reading from database: Database corrupted\r\n```\r\n\r\nSeveral guys from bitcointalk.org faced the same problem too (https://bitcointalk.org/index.php?topic=1288944.0 https://bitcointalk.org/index.php?topic=1281530.0 https://bitcointalk.org/index.php?topic=1152996.0)\r\n\r\nUPDATE: Looks like it's my hardware problem, cancelling bitcoin code claims.",
      "created_at" : "2015-12-17T11:32:15Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-165429785",
      "id" : 165429785,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2015-12-19T15:51:57Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/165429785",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/15796975?v=3",
         "events_url" : "https://api.github.com/users/z0rg1nc/events{/privacy}",
         "followers_url" : "https://api.github.com/users/z0rg1nc/followers",
         "following_url" : "https://api.github.com/users/z0rg1nc/following{/other_user}",
         "gists_url" : "https://api.github.com/users/z0rg1nc/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/z0rg1nc",
         "id" : 15796975,
         "login" : "z0rg1nc",
         "organizations_url" : "https://api.github.com/users/z0rg1nc/orgs",
         "received_events_url" : "https://api.github.com/users/z0rg1nc/received_events",
         "repos_url" : "https://api.github.com/users/z0rg1nc/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/z0rg1nc/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/z0rg1nc/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/z0rg1nc"
      }
   },
   {
      "body" : "I have this issue on a fresh install of debian jessie/sid.\r\n\r\nIt's happening with bitcoin, litecoin and dogecoin.\r\n\r\nIt happens on btrfs, changing filesystem to ext4 and redownloading the blochain helped.",
      "created_at" : "2016-01-10T00:06:43Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-170295547",
      "id" : 170295547,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2016-01-10T01:44:28Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/170295547",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/2027447?v=3",
         "events_url" : "https://api.github.com/users/desci/events{/privacy}",
         "followers_url" : "https://api.github.com/users/desci/followers",
         "following_url" : "https://api.github.com/users/desci/following{/other_user}",
         "gists_url" : "https://api.github.com/users/desci/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/desci",
         "id" : 2027447,
         "login" : "desci",
         "organizations_url" : "https://api.github.com/users/desci/orgs",
         "received_events_url" : "https://api.github.com/users/desci/received_events",
         "repos_url" : "https://api.github.com/users/desci/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/desci/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/desci/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/desci"
      }
   },
   {
      "body" : "I have looked at my logs and this probably has to do with changing the libdb version.\r\n\r\nSee https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=621425",
      "created_at" : "2016-01-11T03:30:11Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-170424153",
      "id" : 170424153,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2016-01-11T03:30:11Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/170424153",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/2027447?v=3",
         "events_url" : "https://api.github.com/users/desci/events{/privacy}",
         "followers_url" : "https://api.github.com/users/desci/followers",
         "following_url" : "https://api.github.com/users/desci/following{/other_user}",
         "gists_url" : "https://api.github.com/users/desci/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/desci",
         "id" : 2027447,
         "login" : "desci",
         "organizations_url" : "https://api.github.com/users/desci/orgs",
         "received_events_url" : "https://api.github.com/users/desci/received_events",
         "repos_url" : "https://api.github.com/users/desci/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/desci/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/desci/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/desci"
      }
   },
   {
      "body" : "I had the same problem on a Mac Mini (mid 2011) running OS X 10.11.2 with all updates installed, 8 GB RAM. It was consistent; neither -reindex nor deleting the entire blockchain and re-downloading would get around the corrupted database problem, no matter how many times I tried. However, replacing the 0.11.2 software with 0.10.3 (64 bit) worked, after I ran the 0.10.3 with -reindex. So there is something in 0.11.2 that doesn't like my machine, but I can work with 0.10.3 for now.",
      "created_at" : "2016-01-15T17:12:51Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-172018945",
      "id" : 172018945,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2016-01-15T17:12:51Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/172018945",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/1759745?v=3",
         "events_url" : "https://api.github.com/users/jloughry/events{/privacy}",
         "followers_url" : "https://api.github.com/users/jloughry/followers",
         "following_url" : "https://api.github.com/users/jloughry/following{/other_user}",
         "gists_url" : "https://api.github.com/users/jloughry/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/jloughry",
         "id" : 1759745,
         "login" : "jloughry",
         "organizations_url" : "https://api.github.com/users/jloughry/orgs",
         "received_events_url" : "https://api.github.com/users/jloughry/received_events",
         "repos_url" : "https://api.github.com/users/jloughry/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/jloughry/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/jloughry/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/jloughry"
      }
   },
   {
      "body" : "We are having the same problem and it is recurring. All other clients seem fine (Litecoin, Dogecoin, Unobtanium etc) - it seems to only be affecting bitcoin.\r\n\r\n2016-01-22 16:30:51 Leaving block file 424: CBlockFileInfo(blocks=0, size=0, heights=0...0, time=1970-01-01...1970-01-01)\r\n2016-01-22 16:30:56 Pre-allocating up to position 0x1000000 in blk00424.dat\r\n2016-01-22 16:30:56 LevelDB read failure: Corruption: block checksum mismatch\r\n2016-01-22 16:30:56 Corruption: block checksum mismatch\r\n2016-01-22 16:30:56 Error: Error reading from database, shutting down.\r\n2016-01-22 16:30:56 Error reading from database: Database corrupted\r\n\r\n\"version\" : 110200,\r\n \"protocolversion\" : 70002,\r\n \"walletversion\" : 60000,\r\n",
      "created_at" : "2016-01-22T16:37:46Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-173971360",
      "id" : 173971360,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2016-01-22T16:38:43Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/173971360",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/4410912?v=3",
         "events_url" : "https://api.github.com/users/ethought/events{/privacy}",
         "followers_url" : "https://api.github.com/users/ethought/followers",
         "following_url" : "https://api.github.com/users/ethought/following{/other_user}",
         "gists_url" : "https://api.github.com/users/ethought/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/ethought",
         "id" : 4410912,
         "login" : "ethought",
         "organizations_url" : "https://api.github.com/users/ethought/orgs",
         "received_events_url" : "https://api.github.com/users/ethought/received_events",
         "repos_url" : "https://api.github.com/users/ethought/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/ethought/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/ethought/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/ethought"
      }
   },
   {
      "body" : "I have tried changing libdb versions on a debian sid machine and only bitcoin is being affected as of now. Dogecoin and every other wallet are fine.",
      "created_at" : "2016-01-23T01:01:19Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-174101261",
      "id" : 174101261,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2016-01-23T01:01:19Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/174101261",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/2027447?v=3",
         "events_url" : "https://api.github.com/users/desci/events{/privacy}",
         "followers_url" : "https://api.github.com/users/desci/followers",
         "following_url" : "https://api.github.com/users/desci/following{/other_user}",
         "gists_url" : "https://api.github.com/users/desci/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/desci",
         "id" : 2027447,
         "login" : "desci",
         "organizations_url" : "https://api.github.com/users/desci/orgs",
         "received_events_url" : "https://api.github.com/users/desci/received_events",
         "repos_url" : "https://api.github.com/users/desci/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/desci/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/desci/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/desci"
      }
   },
   {
      "body" : "Changing libdb won't fix this: the backing database for the utxo set and block index is leveldb, not berkeleydb.",
      "created_at" : "2016-01-28T11:32:32Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-176134844",
      "id" : 176134844,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2016-01-28T11:32:32Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/176134844",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/126646?v=3",
         "events_url" : "https://api.github.com/users/laanwj/events{/privacy}",
         "followers_url" : "https://api.github.com/users/laanwj/followers",
         "following_url" : "https://api.github.com/users/laanwj/following{/other_user}",
         "gists_url" : "https://api.github.com/users/laanwj/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/laanwj",
         "id" : 126646,
         "login" : "laanwj",
         "organizations_url" : "https://api.github.com/users/laanwj/orgs",
         "received_events_url" : "https://api.github.com/users/laanwj/received_events",
         "repos_url" : "https://api.github.com/users/laanwj/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/laanwj/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/laanwj/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/laanwj"
      }
   },
   {
      "body" : "FYI: I had the same issue with v0.11.0 on AMD64 Debian Jessie Linux with ext4 on LUKS file system (Bitcoin-Qt compiled by me).\r\n\r\nI ran memtest86 and found a couple of faults in my RAM. I disabled 2 sections of 64kB of misbehaving RAM in the GRUB configuration, restarted, and started re-indexing with Bitcoin. Re-indexing took almost a week on my computer; every 8 hours or so I stopped Bitcoin, made a backup of the databases and restarted(*) Bitcoin, just in case something went wrong halfway or at 99%, to prevent having to start re-indexing again. However, I did not encounter any problems during re-indexing, so for me, the problem is solved.\r\n\r\nIf you are always encountering issues during re-indexing on the same position in the block chain, could it be that there is something wrong in the block files stored on your hard disk? In that case, would it be possible to resolve the issue by removing a certain range of block files, to force Bitcoin to re-download these blocks? WARNING: this is just guess-work, and certainly not a recommendation! I'm not that much of an expert, and from my POV this might as well make things worse.\r\n\r\n(*) On 2nd, 3rd etc. start of Bitcoin, I did not provide the re-index commandline argument, since I did not want to re-start indexing every time. Apparently, Bitcoin automatically continues re-indexing if re-indexing was unfinished on the last shutdown.\r\n",
      "created_at" : "2016-01-29T23:22:54Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-177014517",
      "id" : 177014517,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2016-01-29T23:22:54Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/177014517",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/2883848?v=3",
         "events_url" : "https://api.github.com/users/cornwarecjp/events{/privacy}",
         "followers_url" : "https://api.github.com/users/cornwarecjp/followers",
         "following_url" : "https://api.github.com/users/cornwarecjp/following{/other_user}",
         "gists_url" : "https://api.github.com/users/cornwarecjp/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/cornwarecjp",
         "id" : 2883848,
         "login" : "cornwarecjp",
         "organizations_url" : "https://api.github.com/users/cornwarecjp/orgs",
         "received_events_url" : "https://api.github.com/users/cornwarecjp/received_events",
         "repos_url" : "https://api.github.com/users/cornwarecjp/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/cornwarecjp/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/cornwarecjp/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/cornwarecjp"
      }
   },
   {
      "body" : "Reindexing with -par=1 solved it for me on MacOS running 0.11.2.  \r\n\r\nThe only interesting thing my side is that when I had the error for the first time I had just finished re-downloading the whole chain using 0.11.2, after upgrading from 0.11.1. I did not need to re-download, I just did it (it is not relevant why). This would seem to suggest it is a problem that - at least on Mac - arose from 0.11.2.\r\n\r\nFrom people's descriptions though it seems to me we may be in front of more than one issue. The range of symptoms is wide and inconsistent.\r\n\r\nG.",
      "created_at" : "2016-01-30T06:39:09Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-177087816",
      "id" : 177087816,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2016-01-30T06:39:09Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/177087816",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/303693?v=3",
         "events_url" : "https://api.github.com/users/giacecco/events{/privacy}",
         "followers_url" : "https://api.github.com/users/giacecco/followers",
         "following_url" : "https://api.github.com/users/giacecco/following{/other_user}",
         "gists_url" : "https://api.github.com/users/giacecco/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/giacecco",
         "id" : 303693,
         "login" : "giacecco",
         "organizations_url" : "https://api.github.com/users/giacecco/orgs",
         "received_events_url" : "https://api.github.com/users/giacecco/received_events",
         "repos_url" : "https://api.github.com/users/giacecco/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/giacecco/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/giacecco/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/giacecco"
      }
   },
   {
      "body" : "Same problem. 0.12.1  Win7 x64\r\n\r\n2016-04-21 22:05:08 Corruption: block checksum mismatch\r\n2016-04-21 22:05:08 *** System error while flushing: Database corrupted\r\n2016-04-21 22:19:48 UPnP Port Mapping successful.\r\n2016-04-21 22:22:15 ERROR: ProcessNewBlock: ActivateBestChain failed\r\n2016-04-21 22:22:15 tor: Thread interrupt\r\n2016-04-21 22:22:15 opencon thread interrupt\r\n2016-04-21 22:22:15 net thread interrupt\r\n2016-04-21 22:22:15 torcontrol thread exit\r\n2016-04-21 22:22:15 addcon thread interrupt\r\n2016-04-21 22:22:15 scheduler thread interrupt\r\n2016-04-21 22:22:15 msghand thread interrupt\r\n2016-04-21 22:22:15 Shutdown: In progress...\r\n2016-04-21 22:22:15 StopNode()\r\n2016-04-21 22:22:16 UPNP_DeletePortMapping() returned: 0\r\n2016-04-21 22:22:16 upnp thread interrupt\r\n2016-04-21 22:22:16 Corruption: block checksum mismatch\r\n2016-04-21 22:22:16 *** System error while flushing: Database corrupted\r\n2016-04-21 22:22:18 Shutdown: done",
      "created_at" : "2016-04-21T22:54:47Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-213150716",
      "id" : 213150716,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2016-04-21T22:54:47Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/213150716",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/247089?v=3",
         "events_url" : "https://api.github.com/users/lostcodder/events{/privacy}",
         "followers_url" : "https://api.github.com/users/lostcodder/followers",
         "following_url" : "https://api.github.com/users/lostcodder/following{/other_user}",
         "gists_url" : "https://api.github.com/users/lostcodder/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/lostcodder",
         "id" : 247089,
         "login" : "lostcodder",
         "organizations_url" : "https://api.github.com/users/lostcodder/orgs",
         "received_events_url" : "https://api.github.com/users/lostcodder/received_events",
         "repos_url" : "https://api.github.com/users/lostcodder/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/lostcodder/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/lostcodder/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/lostcodder"
      }
   },
   {
      "body" : "I have the same problem :(\r\nKubuntu 16.04 64bit\r\nbitbitcoin-qt 0.12.1.0-g9779e1e\r\n\r\n~~~~~~~~~~~~\r\n2016-05-16 05:40:43 Corruption: block checksum mismatch\r\n2016-05-16 05:40:43 *** System error while flushing: Database corrupted\r\n2016-05-16 05:57:02 ping timeout: 1200.046503s\r\n2016-05-16 05:57:23 ping timeout: 1200.016167s\r\n2016-05-16 05:59:52 ping timeout: 1200.020610s\r\n2016-05-16 07:33:18 ERROR: ProcessNewBlock: ActivateBestChain failed\r\n2016-05-16 07:33:18 tor: Thread interrupt\r\n2016-05-16 07:33:18 torcontrol thread exit\r\n2016-05-16 07:33:18 addcon thread interrupt\r\n2016-05-16 07:33:18 opencon thread interrupt\r\n2016-05-16 07:33:18 msghand thread interrupt\r\n2016-05-16 07:33:18 scheduler thread interrupt\r\n2016-05-16 07:33:18 net thread interrupt\r\n2016-05-16 07:33:18 Shutdown: In progress...\r\n2016-05-16 07:33:18 StopNode()\r\n2016-05-16 07:33:18 Corruption: block checksum mismatch\r\n2016-05-16 07:33:18 *** System error while flushing: Database corrupted\r\n2016-05-16 07:33:30 Shutdown: done\r\n~~~~~~~~~~~~\r\n",
      "created_at" : "2016-05-16T07:49:30Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-219368652",
      "id" : 219368652,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2016-05-16T07:49:55Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/219368652",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/16278695?v=3",
         "events_url" : "https://api.github.com/users/netzstrand/events{/privacy}",
         "followers_url" : "https://api.github.com/users/netzstrand/followers",
         "following_url" : "https://api.github.com/users/netzstrand/following{/other_user}",
         "gists_url" : "https://api.github.com/users/netzstrand/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/netzstrand",
         "id" : 16278695,
         "login" : "netzstrand",
         "organizations_url" : "https://api.github.com/users/netzstrand/orgs",
         "received_events_url" : "https://api.github.com/users/netzstrand/received_events",
         "repos_url" : "https://api.github.com/users/netzstrand/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/netzstrand/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/netzstrand/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/netzstrand"
      }
   },
   {
      "body" : "Update: this problem went away when I replaced the memory (RAM) in my computer. The old memory was found to have errors by memtest (data mismatch error). The new memory has no such errors according to memtest.\r\n\r\nBitcoin Core has been running 24/7 for weeks now with no problems on a Mac Mini (Mid 2011, 2.3 GHz Intel Core i5, 16 GB 1600 MHz DDR3 memory, OS X 10.11.4).",
      "created_at" : "2016-05-16T19:21:51Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-219520277",
      "id" : 219520277,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2016-05-16T19:21:51Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/219520277",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/1759745?v=3",
         "events_url" : "https://api.github.com/users/jloughry/events{/privacy}",
         "followers_url" : "https://api.github.com/users/jloughry/followers",
         "following_url" : "https://api.github.com/users/jloughry/following{/other_user}",
         "gists_url" : "https://api.github.com/users/jloughry/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/jloughry",
         "id" : 1759745,
         "login" : "jloughry",
         "organizations_url" : "https://api.github.com/users/jloughry/orgs",
         "received_events_url" : "https://api.github.com/users/jloughry/received_events",
         "repos_url" : "https://api.github.com/users/jloughry/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/jloughry/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/jloughry/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/jloughry"
      }
   },
   {
      "author_association" : "NONE",
      "body" : "~~For what it's worth, I've run Memtest86+ overnight for two nights and found no errors in my RAM, so that's not the only reason this error can arise.~~\n\nDiscovered that I was getting some very occasional single-bit errors when reading from disk, so that was likely the cause of this error for me. I underclocked my system by 3.76%, and now I am unable to reproduce the single-bit errors, so hopefully my \"block checksum mismatch\" issues will be gone now too.\n",
      "created_at" : "2016-05-17T04:17:04Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-219615400",
      "id" : 219615400,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2016-05-19T21:20:19Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/219615400",
      "user" : {
         "avatar_url" : "https://avatars3.githubusercontent.com/u/797782?v=4",
         "events_url" : "https://api.github.com/users/whitslack/events{/privacy}",
         "followers_url" : "https://api.github.com/users/whitslack/followers",
         "following_url" : "https://api.github.com/users/whitslack/following{/other_user}",
         "gists_url" : "https://api.github.com/users/whitslack/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/whitslack",
         "id" : 797782,
         "login" : "whitslack",
         "organizations_url" : "https://api.github.com/users/whitslack/orgs",
         "received_events_url" : "https://api.github.com/users/whitslack/received_events",
         "repos_url" : "https://api.github.com/users/whitslack/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/whitslack/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/whitslack/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/whitslack"
      }
   },
   {
      "author_association" : "NONE",
      "body" : "I had the same error i fixed it by formatting my disk to f2fs from ext4.\n",
      "created_at" : "2016-07-10T00:56:57Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-231564325",
      "id" : 231564325,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2016-07-10T00:57:11Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/231564325",
      "user" : {
         "avatar_url" : "https://avatars1.githubusercontent.com/u/6427921?v=4",
         "events_url" : "https://api.github.com/users/hamiltino/events{/privacy}",
         "followers_url" : "https://api.github.com/users/hamiltino/followers",
         "following_url" : "https://api.github.com/users/hamiltino/following{/other_user}",
         "gists_url" : "https://api.github.com/users/hamiltino/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/hamiltino",
         "id" : 6427921,
         "login" : "hamiltino",
         "organizations_url" : "https://api.github.com/users/hamiltino/orgs",
         "received_events_url" : "https://api.github.com/users/hamiltino/received_events",
         "repos_url" : "https://api.github.com/users/hamiltino/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/hamiltino/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/hamiltino/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/hamiltino"
      }
   },
   {
      "author_association" : "NONE",
      "body" : "This appears to be entirely related to hardware failures. Closeable @borisbontje",
      "created_at" : "2017-09-05T13:39:35Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-327178146",
      "id" : 327178146,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2017-09-05T13:39:35Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/327178146",
      "user" : {
         "avatar_url" : "https://avatars2.githubusercontent.com/u/14876936?v=4",
         "events_url" : "https://api.github.com/users/Leviathn/events{/privacy}",
         "followers_url" : "https://api.github.com/users/Leviathn/followers",
         "following_url" : "https://api.github.com/users/Leviathn/following{/other_user}",
         "gists_url" : "https://api.github.com/users/Leviathn/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/Leviathn",
         "id" : 14876936,
         "login" : "Leviathn",
         "organizations_url" : "https://api.github.com/users/Leviathn/orgs",
         "received_events_url" : "https://api.github.com/users/Leviathn/received_events",
         "repos_url" : "https://api.github.com/users/Leviathn/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/Leviathn/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/Leviathn/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/Leviathn"
      }
   },
   {
      "author_association" : "NONE",
      "body" : "@jorisbontje",
      "created_at" : "2017-09-05T13:40:12Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/6528#issuecomment-327178314",
      "id" : 327178314,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/6528",
      "updated_at" : "2017-09-05T13:40:12Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/327178314",
      "user" : {
         "avatar_url" : "https://avatars2.githubusercontent.com/u/14876936?v=4",
         "events_url" : "https://api.github.com/users/Leviathn/events{/privacy}",
         "followers_url" : "https://api.github.com/users/Leviathn/followers",
         "following_url" : "https://api.github.com/users/Leviathn/following{/other_user}",
         "gists_url" : "https://api.github.com/users/Leviathn/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/Leviathn",
         "id" : 14876936,
         "login" : "Leviathn",
         "organizations_url" : "https://api.github.com/users/Leviathn/orgs",
         "received_events_url" : "https://api.github.com/users/Leviathn/received_events",
         "repos_url" : "https://api.github.com/users/Leviathn/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/Leviathn/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/Leviathn/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/Leviathn"
      }
   }
]
